src/views/Login.vue

```diff
@@ -87,6 +88,7 @@
         @scan="onScanQrcode"
       />
     </container>
+    <ProgressIndicator :show="isProgressIndicatorShow" />
```

Comment:
This is not needed. Remove `ProgressIndicator` and rollback all the changes inside Login.vue